_components/banner.md

```diff
@@ -26,16 +26,16 @@ lead: The banner identifies your site as an official website of the United State
       </ul>
       <h4>When to consider something else</h4>
       <ul class="usa-content-list">
-        <li><strong>If you use neither https nor a .gov/.mil domain.</strong> The banner text calls out https and .gov/.mil domains as ways to communicate that this is an official government site. Don't use the banner if your site doesn't do those things.</li>
-        <li><strong>Any time it would be misleading.</strong> The banner should be used to reduce confusion. Avoid using the banner on sites meant only for testing or experiments, or are otherwise not meant to be seen as an official website.</li>
+        <li><strong>If you use neither HTTPS nor a .gov/.mil domain.</strong> The banner text indentifies HTTPS and .gov/.mil domains as indicators that a website is an official government website. Use the banner only if your site uses both the proper TLD <em>and</em> HTTPS.</li>
```

Comment:
Plain language suggestion plus fixes typo in `identify`

<li><strong>If you don't use .gov/.mil domain and HTTPS.</strong> The banner text identifies .gov/.mil domains and HTTPS as indicators that a website is an official government website. Use the banner only if your site uses both the proper TLD <em>and</em> HTTPS.</li>